docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -152,55 +152,49 @@ Closes all alerts that match the exception's conditions and were generated only
-with Elastic endpoint rule exceptions. To associate rules, when creating or
-editing a rule select the
-<<rule-ui-advanced-params, _Elastic endpoint exceptions_>> option.
+You can also add Endpoint exceptions to rules that are associated with Elastic Endpoint rule exceptions. To associate rules, when creating or editing a rule, select the <<rule-ui-advanced-params, *Elastic Endpoint exceptions*>> option.
 
 [IMPORTANT]
 =============
-Exceptions added to the Elastic Endpoint Security rule affect all alerts sent
+Exceptions added to the Elastic {endpoint-sec} rule affect all alerts sent
 from the Endpoint agent. Be careful not to unintentionally prevent some Endpoint
 alerts.
+
+Additionally, to add an Endpoint exception to the Elastic {endpoint-sec} rule, there must be at least one {endpoint-sec} alert generated in the system. For non-production use, if no alerts exist, you can trigger a test alert using malware emulation techniques or tools such as the Anti Malware Testfile from the https://www.eicar.org/[European Institute for Computer Anti-Virus Research (EICAR)].
 =============
 
-. To add an Endpoint exception via the rule details page:
+. To add an Endpoint exception from the rule details page:
 .. Go to the rule details page (*Detect* -> *Rules*), and then search for and  select the Elastic *Security Endpoint* rule.
 .. Scroll down to the *Trend* histogram and select the *Exceptions* tab.
 .. Click *Add new exception* -> *Add Endpoint exception*.
-. To add an exception via the Alerts table:
+. To add an exception from the Alerts table:
 .. Go to *Detect* -> *Alerts*.
 .. Scroll down to the Alerts table, and from an Elastic Security Endpoint
 alert, click the *More actions* button (*...*), then select *Add Endpoint exception*.
 +
-The *Add Endpoint Exception* window opens (via the Alerts table).
+. The *Add Endpoint Exception* window opens, from either the rule details page or the Alerts table.
 +
 [role="screenshot"]
 image::images/endpoint-add-exp.png[]
 . If required, modify the conditions.
 +
-NOTE: <<ex-nested-conditions>> describes when nested conditions are required.
+NOTE: See <<ex-nested-conditions>> for more information on when nested conditions are required.
 
 . You can select any of the following:
 
 * *Close this alert*: Closes the alert when the exception is added. This option
-is only available when adding exceptions via the Alerts table.
-* *Close all alerts that match this exception, including alerts generated by other rules*:
+is only available when adding exceptions from the Alerts table.
+* *Close all alerts that match this exception and were generated by this rule*:
 Closes all alerts that match the exception's conditions.
 
-. Click *Add Exception*.
-+
-An exception is created for both the detection rule *and* the Elastic Endpoint
-agent.
+. Click *Add Endpoint Exception*. An exception is created for both the detection rule and the Elastic Endpoint agent.
```

Comment:
I _think_ it's the Elastic Endpoint we are referring to instead of the agent - @kevinlog can you confirm? 

```suggestion
. Click *Add Endpoint Exception*. An exception is created for both the detection rule and the {elastic-endpoint}.
```

Reply:
@jmikell821 that is correct.  Endpoint Exceptions are for Endpoints specifically.